src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -565,6 +628,72 @@ void receiveBinaryMessage(VertxTestContext context) throws InterruptedException,
         assertThat(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS), is(true));
     }
 
+    @Test
+    void receiveBinaryMessageWithHeaders(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
I don't know how much it makes sense to have a different test with binary value when it doesn't impact the headers. Wdyt @see-quick ?

Reply:
Given that I updated both [HttpJsonMessageConverter](https://github.com/strimzi/strimzi-kafka-bridge/blob/master/src/main/java/io/strimzi/kafka/bridge/http/converter/HttpJsonMessageConverter.java) and [HttpBinaryMessageConverter](https://github.com/strimzi/strimzi-kafka-bridge/blob/master/src/main/java/io/strimzi/kafka/bridge/http/converter/HttpBinaryMessageConverter.java) testing both cases was a way to increase coverage.

Reply:
Overall, if currently there are two different classes HttpJsonMessageConverter, HttpBinaryMessageConverter with the related logic now it makes sense. I do not know how were tests before but now it looks good.